Projects/Offer/Sources/Components/MainContentForm/MainContentForm.swift

```diff
@@ -47,7 +48,8 @@ extension MainContentForm: Presentable {
 			container.didLayoutSignal,
 			formContainer.didLayoutSignal,
 			form.didLayoutSignal,
-			scrollView.didScrollSignal
+			scrollView.didScrollSignal,
+			state.quotesSignal.toVoid()
```

Comment:
Not an optimal solution, the root cause of the offsetting should probably be identified instead